styles.css

```diff
@@ -1,3 +1,230 @@
 * {
-	box-sizing: border-box;
-}
\ No newline at end of file
+	box-sizing: border-box;	
+}
+
+html {
+	font-family: Georgia, serif;
+	scroll-padding-top: 70px;
+}
+
+
+/*nav bar*/
+nav {
+	background-color: #6B9080;
+	padding: 30px;
+	position: fixed;
+	width: 100%;
+}
+
+.nav-links a {
+	text-decoration: none;
+	margin-left: 20px;
+	color: #A4C3B2;	
+}
+
+/*nav mobile amends*/
+@media screen and (max-width: 768px){
+.links {
+		display: flex;
+		justify-content: center;
+	}
+}
+
+/*common section styling*/
+.overall {
+	width: 80%;
+	/*background-color: red;*/
+}
+
+/*sections with flex box center*/
+header, footer, main, .my-projects {
+	display: flex;
+	justify-content: center;
+} 
+
+/*header*/
+header {
+	background-color: #CCE3DE;
+	padding: 20px;		
+}
+
+.header {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: 80px;
+}
+
+.header p{
+	margin-bottom: 25px;
+}
+
+header img {
+	max-width: 30%;
+	max-height: 30%;
+	margin-right: 20px;
+	border-radius: 80px;
+	padding: 10px;
+}
+
+.header .button {
+	border: 2px solid #6B9080;
+	border-radius: 12px;
+	padding: 10px;
+	background-color: #EAF4F4;
+	cursor: pointer;
+	text-decoration: none;
+}
+
+/*header mobile amends*/
+@media screen and (max-width: 768px){
+.header {
+	flex-wrap: wrap-reverse;
+	text-align: center;
+	}
+header img {
+	padding: 25px 10px 10px 10px;
+}
+}
+
+/*main about me*/
+.about-me {
+	background-color: #F6FFF8;
+	justify-content: center;
+}
+
+.my-details {
+	padding-top: 100px;
+	padding-bottom: 100px;
+}
+
+.my-details p {
+	text-align: justify;
+}
+
+.my-details h2 {
+	text-align: center;
+	margin-top: 20px;
+	padding-top: 20px;
+}
+
+/*porfolio summary*/
+
+.my-projects {
+	background-color: #EAF4F4;
+	padding-top: 100px;
+	padding-bottom: 100px;
+}
+
+.portfolio {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-around;
+	padding: 20px;
+	justify-content: center;
+	padding: 0;
+}
+
+.my-projects h2 {
+	text-align: center;
+	padding-top: 20px;
+}
+
+.my-projects p {
+	text-align: center;
+}
+
+article {
+	/*border: blue dotted 5px;*/
+	display: flex;
+	width: 400px;
+	height: 400px;
+	margin: 10px;
+	background: #CCE3DE;
+	box-shadow: 5px 10px #6B9080;
+}
+
+article > div {
+	width: 50%;
+	/*border: dotted black;*/
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between; 
+}
+
+.portfolio img {
+	width: 90%;
+	height: 50%;
```

Comment:
when you set both the width and the height on an image it runs the risk of breaking the aspect ratio. safer to just set one, usually width.